src/BlockchainTestsFiller/InvalidBlocks/bcEIP1559/badBlocksFiller.yml

```diff
@@ -112,19 +112,23 @@ badBlocks:
   #
   # From the EIP:
   #
-  # assert block.gas_target <= parent_gas_target + parent_gas_target // 1024, 'invalid block: gas target increased too much'
+  # assert block.gas_limit < parent_gas_limit + parent_gas_limit // 1024, 
+  #       'invalid block: gas limit increased too much'
 
   - blockHeader:
-      gasLimit: 1073740802
-    blocknumber: 3
```

Comment:
why removed this explicit blocknumbers ?

Reply:
Oops, I wasn't sure why I did that. Anyway, they're back there now.